analytics/Makefile

```diff
@@ -5,7 +5,9 @@
 ORG ?= HHS
 REPO ?= simpler-grants-gov
 SPRINT_PROJECT ?= 13
+ROADMAP_PROJECT ?= 12
 SPRINT_FILE ?= data/sprint-data.json
+ROADMAP_FILE ?= data/roadmap-data.json
```

Comment:
The step functions can't write to the `data/` directory. You need to change the directory for `ROADMAP_FILE` from `data/roadmap-data.json` in this file: https://github.com/HHS/simpler-grants-gov/blob/main/infra/analytics/service/sfn_sprint_reports.tf#L36

Or, even better, making it resolve to `${OUTPUT_DIR}/roadmap-data.json`, so you don't need to add a new makefile variable into the terraform file at all.

Reply:
Great thanks for flagging this! I'll update

Reply:
Should I update that for the other files too?

Reply:
If you care about consistency, yes.

Reply:
Okay cool, I updated them in this commit: [refactor: Parametrizes output directory in Makefile](https://github.com/HHS/simpler-grants-gov/pull/1764/commits/ba66758efb642b730d5c8264e8b3d58a1a0ff514)